Gruntfile.js

```diff
@@ -8,7 +8,7 @@ module.exports = function (grunt) {
         distdir: "dist",
         srcdir: "src",
         builddir: ".work/.tmp",
-        name: grunt.file.readJSON("package.json").name || "ovh-angular-q-allSettled", // module name
+        name: "ovh-angular-q-allSettled", // module name
```

Comment:
This is a temporarily fix due to difference between package name value and the angular module name.

see: https://github.com/ovh-ux/ovh-angular-q-allsettled/blob/master/package.json#L2